### ISSUE TITLE
Error ofcompilation in  angular 9

### ISSUE THREAD

**Issue by pingouin-creation** [malothnaresh/mat-select-autocomplete#28]
getting an error of compilation:

ERROR in Failed to compile entry-point mat-select-autocomplete (module as esm5) due to compilation errors:   
node_modules/mat-select-autocomplete/fesm5/mat-select-autocomplete.js:282:30 - error NG1010: Value at position 3 in the NgModule.imports of SelectAutocompleteModule is not a reference: [object Object]

**Comment by aish16** [comment]
I am getting the same exception. Is any one working on this ?
It looks like the 2 files are referring to older version of Material components:

Existing:
`import { MatButtonModule, MatCheckboxModule } from '@angular/material';`

Expected:
`
import { MatButtonModule} from '@angular/material/button';
import { MatCheckboxModule} from '@angular/material/checkbox';
`

**Comment by achilehero** [comment]
No, it's just that angular 10 has ivy enabled by default, which supports tree-shaking, so you need to include each component on it's own. I have a PR for that open, so @malothnaresh , maybe you can check it.

Thanks! :)
